### DISCUSSION TITLE
Setting the initial version

### DISCUSSION THREAD

**Discussion by dreads** [PaulHatch/semantic-version#71] [Q&A]
Is there an initial setup of tags in repo with the default behavior of the action? For example, must a tag first be created and pushed to the repository and with a specific format? The documentation states that no one ever has to choose or manually assign the version number.

**Comment by PaulHatch** [comment]
The "default" is 0.0.0, but if you want to assign an initial version, just add the tag to an existing commit before running. This action is predicting/suggesting the next version, but you can skip/override that if you want. The next time the action runs it will pick up whatever the latest tag is and use that as the starting point to provide the next version.

By default the tag has a format of `v{version}`, you can drop the `v` prefix by setting the tag prefix value to an empty string.

Note that if the commit which the action is run against is already tagged (vs a previous commit being tagged), the action will always return that tag.
